docs/for-mentors/tabs-and-useful-links.md

```diff
@@ -0,0 +1,89 @@
+# Tabs and Useful Links
+
+In order to quickly set up for a mentor shift, using browser profiles might be
+useful. This setup should work well both in Chrome and Firefox, as they both
+support user profiles (though it’s easier to use in Chrome as it has a native UI
+to switch between profiles).
+
+During a mentor shift, you’ll need a few things:
+
+* Zulip to monitor questions from students
+* GCI app to monitor the task queue
+* GitHub to monitor PR submissions by students
+* Zulip development environment to fetch and test PRs you’re reviewing
+* links to various helper docs, mainly but not limited to the `zulip-gci` repo,
+  Zulip docs, and GCI website
+
+## Browser setup
+
+This setup allows you to quickly open all the tools you need for the mentor
+shift by opening the browser profile and just as easily end your shift by
+closing it.
+
+1. Create a separate browser profile for GCI - you can easily do it in 
+[Chrome](https://support.google.com/chrome/answer/2364824?co=GENIE.Platform%3DDesktop&hl=en) or
+[Firefox](https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles)
+
+2. Add pinned tabs
+* https://chat.zulip.org
+* https://codein.withgoogle.com/
+* https://github.com/zulip/zulip-gci-submissions
+
+3. Add a tab manager
```

Comment:
The paragraphs in this point need more whitespace to be properly rendered as a part of it (I think it's 3 spaces, at least that's what worked for me in other docs) :)